examples/text-to-speech/sync_tts_transcription.py

```diff
@@ -0,0 +1,168 @@
+import asyncio
+import logging
+from typing import Optional
+
+from livekit import rtc
+from livekit.agents import (
+    JobContext,
+    JobRequest,
+    WorkerOptions,
+    cli,
+    transcription,
+    tts,
+)
+from livekit.plugins import elevenlabs
+
+
+def _text_to_chunks(text: str) -> list[str]:
+    """Split the text into chunks of 2, 3, and 4 words"""
+    sizes = [2, 3, 4]
+    chunks, i = [], 0
+
+    for size in sizes:
+        while i + size <= len(text):
+            chunks.append(text[i : i + size])
+            i += size
+
+    chunks.append(text[i:])  # remaining
+    return chunks
+
+
+async def _playout_task(
+    playout_q: asyncio.Queue, audio_source: rtc.AudioSource
+) -> None:
+    """Playout audio frames from the queue to the audio source"""
+    while True:
+        frame = await playout_q.get()
+        if frame is None:
+            break
+
+        await audio_source.capture_frame(frame)
+
+
+async def entrypoint(job: JobContext):
+    logging.info("starting transcription protocol example")
+
+    tts_11labs = elevenlabs.TTS()
+
+    # publish an audio track
+    source = rtc.AudioSource(tts_11labs.sample_rate, tts_11labs.num_channels)
+    track = rtc.LocalAudioTrack.create_audio_track("agent-mic", source)
+    options = rtc.TrackPublishOptions(source=rtc.TrackSource.SOURCE_MICROPHONE)
+    await job.room.local_participant.publish_track(track, options)
+
+    #
+    # 1. example using a tts stream (we split text into chunks just for the example)
```

Comment:
these are separate examples, can we put them in three separate functions?
i.e. 
* streaming multiple segments
* single_segment
* deferred playout